### ISSUE TITLE
Extract command breaks translation files

### ISSUE THREAD

**Issue by emzet** [ngneat/transloco-keys-manager#32]
## I'm submitting a...
```
[X] Bug report
```

## Current behavior
When runing `transloco-keys-manager extract` with this template markup, it outputs message in console and breaks existing translation JSON files.

**Bad**
```
<a [routerLink]="'MY_MODULE' | lowercase"
  [title]="'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco">
  {{ 'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}
</a>
```

**Good** (_no message in console, extract successful_)
```
<a [title]="'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco"
  [routerLink]="'MY_MODULE' | lowercase">
  {{ 'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}
</a>
```

So it depends on some order of element attributes or something else. Maybe issue with any other HTML elements or attributes.

## Expected behavior
Extract translation keys without breaking the existing translation files.

## Minimal reproduction of the problem with instructions
For example add to any template:
```
<a [routerLink]="'MY_MODULE' | lowercase"
  [title]="'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco">
  {{ 'MY_MODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}
</a>
```

Then run from console `transloco-keys-manager extract` and see message in console and changes on your existing translation JSON files (defined in `transloco.config.js`).

## What is the motivation / use case for changing the behavior?
To work `transloco-keys-manager extract` as intended and without breaking the existing translation files.

## Environment

```
Angular version: 9.1.3
 
For Tooling issues:
- Node version: v14.0.0
- Platform: Windows 10
```

**Comment by shaharkazaz** [comment]
What do you mean by breaking? Can you provide an example?

**Comment by emzet** [comment]
It removes existing translation keys from files e.g. under `MY_MODULE` group and it outputs message to the console: `The following keys won't be accessible when unflatting the object:` and there is a list of  translation keys, which were removed. It preserves only key `"MY_MODULE": "MY_MODULE"` 

_(transloco.config.js setting `defaultValue` is set to  `'{{key}}'`, so there the value is same as key)_

**Comment by shaharkazaz** [comment]
@emzet Can you please verify this is resolved in v2.0.6? 🙂
